isis/src/base/objs/Pipeline/Pipeline.truth

```diff
@@ -1,94 +1,94 @@
 Simple Pipe
 PIPELINE -------> unitTest2 <------- PIPELINE
-thm2isis FROM="$ISIS3DATA/odyssey/testData/I00831002RDR.even.cub" TO="/work1/out.cub"
+thm2isis FROM="$odyssey/testData/I00831002RDR.even.cub" TO="/work1/out.cub"
```

Comment:
How is this full path on the TO parameter working?

Reply:
Huh, I also have this question now.

Reply:
This isn't a real run. That's why it works.